build-logic/android/src/main/kotlin/convention.android-robolectric.gradle.kts

```diff
@@ -0,0 +1,37 @@
+import com.android.build.gradle.BaseExtension
+import com.avito.android.withVersionCatalog
+
+plugins {
+    id("convention.android-base")
+}
+
+configure<BaseExtension> {
+    val sharedTestSourcesFolder = "src/sharedTest/kotlin"
```

Comment:
So tests should be placed there?

Reply:
You can run tests that are placed there both with an emulator and without it.

Reply:
Did you check that these tests still runs on emulators on PR?

Reply:
Tests on emulators are broken now. I've created a bug, see MBS-12302.